packages/ui-tabs/src/Tabs/index.tsx

```diff
@@ -328,9 +328,12 @@ class Tabs extends Component<TabsProps, TabsState> {
     if (typeof this.props.onRequestTabChange === 'function') {
       this.props.onRequestTabChange(event, { index, id })
     }
-
-    this.state.withTabListOverflow &&
-      this.showActiveTabIfOverlayed(this._tabList!.querySelector(`#tab-${id}`))
+    setTimeout(() => {
```

Comment:
This is very ugly :( 
If you put in such code please leave a detailed comment why this hack was necessary